networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -91,7 +89,7 @@ def bind_port(self, context):
         host = common.get_host_from_profile(context.current.get('binding:profile'), context.host)
 
         LOG.debug("Using binding host %s for binding port %s", host, port['id'])
-        hostgroup_name, hostgroup = ACI_CONFIG.get_hostgroup_by_host(host)
+        hostgroup_name, hostgroup = ACI_CONFIG.get_hostgroup_by_host(context._plugin_context, host)
```

Comment:
Maybe add a property to the context object, to allow access to the private member and document in the context class that other components access this attribute